script for building release zip for windows

kaoh opened the pull request:
This scripts build a zip file which builds a Windows distribution. The steps how to do this are not described in the upstream repo. This scripts is also packaging otherwise missing icons.

tiker commented:
Well, I got a prompt about the .cardpeek directory not existing and asking I I wanted to create it.  I selected Yes.  Then I get the following error:
![image](https://user-images.githubusercontent.com/1292866/140068802-e6aaac69-1d03-486a-913a-fd4a2b7e3443.png)

It did create the directory with the .tar.gz file inside.  I'm guessing the issue is that Windows 7 doesn't have native support for .tar.gz archives.  I do have 7zip installed and have .tar.gz extensions associated with it but that may not help.

tiker commented:
I should probably mention, my previous comment was testing the 0.8.5-beta build you have linked to above and not something I built myself.

kaoh commented:
I have updated this version. Now it is checked if  7zip is installed in case tar is not available. I have updated the binary: https://github.com/kaoh/cardpeek/releases/tag/0.8.5-beta
